### DISCUSSION TITLE
Welcome to QRCoder Discussions!

### DISCUSSION THREAD

**Discussion by codebude** [Shane32/QRCoder#267] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ZhuravNI** [comment]
Raffael, Hi! 
Please tell me if it is possible to set the type (home, work, etc.) of the address when working with vCard 3? 
Everything works perfectly, except that it is not possible to change the address type...please help!))

**Comment by harshpandey0** [comment]
hey all i am new to this open source can anyone tell me which issues i have to pickup being as a beginner.

i would to love to connect with you all

LinkedIn : https://www.linkedin.com/in/harsh-pandey-2161hp
Skype : https://join.skype.com/invite/w7Fv2iEpdwlS
ShowCase : https://www.showwcase.com/harshpandey9079939
Twitter : https://twitter.com/harshpandey2121
Github : https://github.com/harsh2161

Thankyou
Harsh Pandey
[harshpandey9079@gmail.com](mailto:harshpandey9079@gmail.com)
